client/lib/common/src/data/resource/resource-store.js

```diff
@@ -1,5 +1,6 @@
 import {Store} from 'flummox';
 import _m from 'mori';
+import _ from 'common/src/lodash.custom';
```

Comment:
Just fyi, I'll probably change this to `lodash` later and do a module replacement in the webpack config to insert `common/src/lodash.custom`. The components should not care whether they get a custom build or not.